api-tests/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "api-tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "api-tests": "vitest --config vitest.config.ts $VERSION",
+    "lint": "eslint tests/**/*.ts",
+    "compile": "tsc -p ./tsconfig.json --noEmit --skipLibCheck",
+    "gen": "graphql-codegen"
+  },
+  "author": "",
+  "license": "ISC",
```

```suggestion
  "license": "BSD",
```

I would keep the same license as this repo